Failed test on build

There are tests failling, please see below:
 
[ERROR]   DocumentControllerIntegrationTest.saveDocument:80->assertThatDocumentNodeTreeIsGeneratedCorrectly:136->assertDocumentNodeContent:220 
Expecting:
  <"Sample document introduction that can be on one line or on multiplelines">
to be equal to:
  <"multiple lines  Sample document introduction that can be on one line or on multiple lines  ">

I think I may have found the issue.

```
java.lang.AssertionError: 
Expecting:
  <"Sample document introduction that can be on one line or on multiple lines">
to be equal to:
  <"multiple lines  Sample document introduction that can be on one line or on multiple lines  ">
ignoring whitespace differences
```

The file `test/resources/pdf/sample_legal_document.pdf` contains:

---

**Document title**

**on**

**multiple lines**

Sample document introduction that can be on one line or

on multiple

lines

---

--- a/src/main/java/com/code4ro/legalconsultation/service/impl/pdf/parser/DocumentNodeParser.java
+++ b/src/main/java/com/code4ro/legalconsultation/service/impl/pdf/parser/DocumentNodeParser.java
@@ -102,7 +102,7 @@ public abstract class DocumentNodeParser {
     }
 
     private String getCurrentLine() {
-        return lines[metadata.getCurrentLineIndex()].trim() + " ";
+        return lines[metadata.getCurrentLineIndex()].trim();
     }
 
     private void skipTitleLines(final String title) {

